charts/mlsstats/templates/cronjob.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: {{ include "batch.apiVersion" . }}
```

Comment:
I don't see S3 creds being set as env vars, how does this work?

Reply:
Good point, we need to set them the same way as we set them for prod. Will check and fix.